modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerPruneImage.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.gradle.plugins.workspace.docker;
+
+import com.bmuschko.gradle.docker.tasks.image.DockerExistingImage;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.PruneCmd;
+import com.github.dockerjava.api.model.PruneType;
+
+import org.gradle.api.logging.Logger;
+import org.gradle.api.provider.Property;
+
+/**
+ * @author Seiphon Wang
+ */
+public class DockerPruneImage extends DockerExistingImage {
```

Comment:
@Seiphon  looks good.  Can you add a test?

Reply:
OK Greg, I will add it and resend the pr